backend/config/locales/fr.yml

```diff
@@ -1,21 +1,21 @@
 fr:
   enrollment_errors:
     copy_enrollment_is_not_validated_nor_refused:
-      title: "L’habilitation initiale est encore en traitement"
-      message: "L’action que vous essayez de faire ne sera possible que lorsque l’habilitation sera validée ou refusée."
+      title: 'L’habilitation initiale est encore en traitement'
```

Comment:
I think that your prettier convert automatically all the double quote in single quote. 
Even if it does not make much difference (except for string interpolation), it will be more consistent to keep double quote here because we mostly use double quote everywhere in the backend. 

Plus, it adds a better readability between the single quote and the `’` character.